app/controllers/assignments_controller.rb

```diff
@@ -6,7 +6,9 @@ class AssignmentsController < ApplicationController
   before_action :set_group
   before_action :check_access, only: %i[edit update destroy reopen]
   after_action :check_reopening_status, only: [:update]
-  after_action :allow_iframe_lti, only: %i[show]
+  if Flipper.enabled?(:lms_integration)
```

Comment:
You have to pass this condition as a block, otherwise you will have to restart the server to take this effect

Reply:
how to do that can you suggest ?

Reply:
done